### DISCUSSION TITLE
Websocket Connections

### DISCUSSION THREAD

**Discussion by Samitha10** [flawiddsouza/Restfox#296] [Q&A] [answered]
When new messages are receiving restfox message list is not auto scrolling for newest messages.

**Comment by flawiddsouza** [comment]
Hi @Samitha10, is the issue resolved? Since you marked this as resolved. The auto scrolling should be working. Can you tell me which version you're on and which operation system you're using?

**Reply by Samitha10** [reply, marked as answer]
Yes issue is solved, I'm experienced some what like this. When websocket is connected it is automatically scrolling for every new messages, if you trying to scroll it manually when messages are receiving it is stop auto scrolling.

**Reply by flawiddsouza** [reply]
Ah I see. Thanks for the update.
